content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Configure Liveness and Readiness Probes
+title: Configure Liveness, Readiness and Startup Probes
```

Comment:
@ebriand:

this PR
/lgtm
however, would you be willing to revise line 9 as well?

Reply:
Done :)